### PR TITLE
Issue 63 - Remove dependency on five.grok

### LINKED ISSUE #63: Remove dependency on five.grok

### PR DESCRIPTION
**Pull request by rodfersou**
closes #63

### CODE REVIEW

File: src/sc/embedder/content/embedder.py [hunk 1]
```diff
@@ -445,7 +444,5 @@ def get_player_pos_class(self):
         return css_class
 
 
-class EmbedderVideoJS(grok.View):
-    grok.context(Interface)
-    grok.require('zope2.View')
-    grok.name('embedder_videojs')
+class EmbedderVideoJS:
```
Comment (rodfersou) [inline src/sc/embedder/content/embedder.py]: note: inherit BrowserView class here

**Comment by rodfersou** [comment]
@hvelarde I think the basic changes are done

**Comment by hvelarde** [comment]
@rodfersou please test it manually and let me know the results.

File: test-4.3.x.cfg [hunk 2]
```diff
@@ -6,3 +6,6 @@ extends =
     https://raw.githubusercontent.com/collective/collective.cover/1.1b1/versions-4.3.x.cfg
 
 test-eggs = collective.cover==1.1b1
+
+[versions]
+plone.namedfile = 3.0.7
```
Comment (hvelarde) [inline test-4.3.x.cfg]: this is not needed.

File: test-4.2.x.cfg [hunk 3]
```diff
@@ -8,3 +8,6 @@ extends =
 test-eggs =
     collective.cover==1.1b1
     Pillow
+
+[versions]
+plone.behavior = 1.1
```
Comment (hvelarde) [inline test-4.2.x.cfg]: why do we need this?

File: test-4.3.x.cfg (hunk 2 above)
Comment (rodfersou) [inline test-4.3.x.cfg]: Travis complain about this.

File: src/sc/embedder/tests/test_content.py [hunk 4]
```diff
@@ -151,8 +148,8 @@ def test_vimeo_oembed(self):
             add_form.widgets['IDublinCore.title'].value, u'The Backwater Gospel')
         self.assertNotEqual(
             add_form.widgets['IDublinCore.description'].value, u'')
-        self.assertEqual(add_form.widgets['width'].value, u'1280')
-        self.assertEqual(add_form.widgets['height'].value, u'720')
+        self.assertEqual(add_form.widgets['width'].value, u'480')
```
Comment (rodfersou) [inline src/sc/embedder/tests/test_content.py]: from time to time vimeo api behaves different.. we already change it other times.. I don't know why..

File: src/sc/embedder/content/configure.zcml [hunk 5]
```diff
@@ -0,0 +1,58 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:browser="http://namespaces.zope.org/browser"
+    xmlns:z3c="http://namespaces.zope.org/z3c">
+
+  <adapter factory=".embedder.EmbedderImageFieldWidget" />
+
+  <z3c:widgetTemplate
```
Comment (hvelarde) [inline src/sc/embedder/content/configure.zcml]: why do we need this? can we get rid of this adapter after merging this PR?

Comment (rodfersou) [inline src/sc/embedder/content/configure.zcml]: I don't know why we need this.. this code was already there..

Comment (rodfersou) [inline src/sc/embedder/content/configure.zcml]: how about to create a new issue to check this?

**Pull request merged**